city_scrapers/settings/__init__.py

```diff
@@ -0,0 +1,15 @@
+import logging
+import os
+
+# https://jansonh.github.io/scrapinghub-gcs/
+# If "GOOGLE_APPLICATION_CREDENTIALS" is the credential rather than a path,
+# we need to write a local file with the credential.
+path = "{}/google-cloud-storage-credentials.json".format(os.getcwd())
```

Comment:
I see that it's mentioned as being in `__init__.py` in the article, but does it have to be here? It seems like it could be more explicit to put it in the commented-out section for GCS

Reply:
Moved to `prod.py` and added new comment to be clear about uncommenting.